documentation/guidance/credential-sharing.md

```diff
@@ -6,13 +6,15 @@ The DevOps and Infosec teams maintain a master account to store, manage and shar
 
 :warning: Credentials should **never** be shared outside the password manager: please **do not** send passwords in plaintext by any other means.
 
-## Getting started
+## Getting an account
 
-Just create an account request from the Slack shortcut menu (:zap: on the bottom-left corner) and you'll be added to the 1Password team with access to the shared secrets you're allowed to use; if you already have a 1Password account, you can use the same email address when requesting access. Your request will need to be approved by [@admin-group](https://helpfulengineering.slack.com/admin/user_groups) unless you already belong to it.
+1. Make a request from the Slack shortcut menu (:zap: on the bottom-left corner) and specify which vaults and secrets you're going to use. If you already have a personal account, you can use the same address for both. 
 
-![:zap: shortcut menu](/images/1password-account-request.png)
+    ![:zap: shortcut menu](/images/1password-account-request.png)    
 
-Once you receive an email from 1Password, you can proceed to set up your browser and devices following [this guide](https://support.1password.com/explore/team-member/)
+2. Your request will be manually triaged and approved by an authorized administrator from [#skill-software-devops](https://helpfulengineering.slack.com/archives/CV54M16QH). Please note that this process is not instantaneous and depends on timezones and human availability.
```

Comment:
```suggestion
2. Your request will be manually triaged and approved by an authorized administrator from [#skill-software-devops](https://helpfulengineering.slack.com/archives/CV54M16QH).

    Please note that this process is not instantaneous and may take up to 24 hours because of timezone differences and other human factors. Duplicate requests and at-mentioning people in Slack will not elicit a faster approval, though you can expedite the process by bringing the pertaining authorizations in advance to [#skill-software-devops](https://helpfulengineering.slack.com/archives/CV54M16QH).
```

Sorry for the late reply, it looks like I was editing it on a browser tab that got lost a few hours ago. I've updated it to include your suggestions, though wording might have room for clarity and style improvements.